File: Trabajo#2/C++/Trabajo#8.cpp
#include <iostream>

using namespace std;

int main(){
	
	double numero, numero1, numero2;
	int n1, n2, n3,n4;
	 
	cout<<"Inserte el valor unitario: " <<endl;
	cin>>n1;
		cout<<"Inserte la cantidad: " <<endl;
	cin>>n2;
	 
	numero = n1*n2; 

if(numero<20000){
		numero1= numero*10/100;
	cout<<"Tiene descuento del 15%: " <<endl;
	cout<<"su valor total es: "<< numero<<endl;
}
else{
	cout<<"no tiene descuento: "<<endl;
	cout<<"Gracias por su compra: "<<endl;
	
}
if(numero>20000){
		numero1= numero*35/100;
	cout<<"Tiene descuento del 35%: " <<endl;
	cout<<"su valor total es: "<< numero1<<endl;
}
else{
	cout<<"no tiene descuento: "<<endl;
	cout<<"Gracias por su compra: "<<endl;
}

}

File: Trabajo#2/Python/Trabajo#8.py
n1 = int(input("ingrese el valor total: "))
n2 = int (input("Ingrese la cantidad: "))

valortotal= n1*n2
if (valortotal<20000)*15/100:
    print("Tiene descuento del 15%")
    print("El valor total es: ",valortotal*15/100)
elif (valortotal>=350000)*35/100:
    print ("El valor total es: ",valortotal*35/100)
else:
    print("El valor total es: ",valortotal*35/100)






